nexus_visualization/src/WorkcellStateVisualizer.hpp

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Rename to `workcell_state_visualizer.*pp` to match rest of codebase.

Reply:
https://github.com/osrf/nexus/pull/42/commits/64abc78ad8a1bafb51d64d76f09689bd9a4e736b